keyboards/handwired/dactyl_manuform/5x6/keymaps/cfarvidson/config.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#define MASTER_LEFT
+
+#define XXX KC_NO
+
+#define LAYOUT_cfarvidson_wrapper(\
```

Comment:
This should go into your keymap, as it's only needed there.

Reply:
I'm really showing my c skills with this question, but will that not collide if I build another keyboard than the dactyl manuform? (If I add it in `users/cfarvidson/cfarvidson.h` that is)

For the ergodox i created `layouts/community/ergodox/cfarvidson/config.h` and `layouts/community/split_3x6_3/cfarvidson/config.h` with that wrapper layout. But the dactyl manuforms layout was not represented in the `layouts/community`.

Reply:
No worries!

The config.h is included in every C file by how qmk has make configured.  So only defines that should be everywhere (such as config settings) should go here. 

Stuff like the keycode define and layout defines shouldn't go here.

Also, technically, the master left isn't needed either, as that is the default behavior.

Reply:
I really don't know how to fix this. It the problem just with this file in this file?

I basically did the same thing as 
- https://github.com/cfarvidson/qmk_firmware/blob/master/keyboards/handwired/dactyl_manuform/5x6/keymaps/manna-harbour_miryoku/config.h  
- https://github.com/cfarvidson/qmk_firmware/blob/master/keyboards/handwired/dactyl_manuform/5x6/keymaps/manna-harbour_miryoku/keymap.c

I must be missing something. 😊 

And I'm able to build `ergodox_ez`, `crkbd` and `tbkmini` by using layouts that are available in the `layouts/community`  like `layouts/community/ergodox/cfarvidson/config.h`

Thank you again for looking at this!